doc/rst/technotes/gpu.rst

```diff
@@ -246,6 +246,45 @@ executable. When the program is executed, the compute capability best suited
 for the available GPU will be loaded by the CUDA runtime. Support for this
 feature for AMD GPUs is planned, but not currently available.
 
+GPU Attributes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Chapel's GPU support makes use of attributes (see `Attributes in Chapel <./attributes.html>`_)
+to control various aspects of how code is compiled or executed on the GPU.
+Specifically, the two GPU-specific Chapel attributes are ``@assertOnGpu``
+(described in `Diagnostics and Utilities`_) and ``@gpu.blockSize``. Because
+Chapel's GPU support primarily works by converting eligible loops into GPU
+kernels, GPU-specific attributes primarily apply to loops. The following
+example demonstrates these attributes:
+
+.. code-block:: chapel
+
+   config const myBlockSize = 128;
+
+   on here.gpus[0] {
+     @assertOnGpu
+     @gpu.blockSize(myBlockSize)
+     foreach i in 1..1024 { /* ... your code here ... */ }
+   }
+
+In the above code, ``@assertOnGpu`` ensures that the ``foreach`` loop is
+GPU-eligible, and ``@gpu.blockSize`` sets the block size for the kernel to
+``myBlockSize`` (128 by default).
```

Comment:
I am slightly worried that this parenthetical can be seen as "128 is the default block size". Maybe dropping it is better?